src/Engine/ProtoScript/Utils/ProtoScriptUtils.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using ProtoCore;
+
+namespace ProtoScript.Utils
+{
+    public static class ProtoScriptUtils
+    {
+        public static ProtoCore.Core CreateCore(Options coreOptions,ProtoCore.Core runnerCore)
```

Comment:
Having static factories in utility classes isn't really ideal. Please catch up with @junmendoza about this - if you need a further discussion I'll be happy to help